udf/int_to_hex_string.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TEMP FUNCTION
+  udf_int_to_hex_string(value INT64) AS ((
+    SELECT STRING_AGG(
+        SPLIT('0123456789ABCDEF','')[OFFSET((value >> (bits*4)) & 0xF)], ''
+        ORDER BY bits DESC
+    )
+    FROM UNNEST(generate_array(0, 91)) AS bits
```

Comment:
an INT64 only contains 16 nibbles, not 92, also I think `nibble` is a more accurate name than `bits` here:
```suggestion
    FROM UNNEST(generate_array(0, 15)) AS nibble
```

Reply:
i see that this is being used to test `udf_zeroed_364_days_active_1_bytes`, which is great, but this function shouldn't return extra bytes because it would cause unnecessary confusion for people accessing our library of UDFs.